pipelines/mitx/mfe-frontend-app-learning.yml

```diff
@@ -0,0 +1,65 @@
+---
+resource_types:
+    - name: rclone
+      type: docker-image
+      source:
+          repository: sothr/concourse-rclone-resource
+          tag: latest
+
+resources:
+  - name: mfe-app-learning
+    type: git
+    source:
+      uri: https://github.com/edx/frontend-app-learning.git
+      branch: master
+  - name: mfe-app-learning-bucket
+    type: rclone
+    source:
+      config: |
+        [s3-remote]
+        type = s3
+        provider = AWS
+        env_auth = true
+        region = us-east-1
+
+jobs:
+  - name: build
+    plan:
+      - get: mfe-app-learning
+        trigger: true
+      - task: npm-install-and-build
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: node
+          inputs:
+            - name: mfe-app-learning
+          outputs:
+            - name: mfe-app-learning/dist
+          run:
+            path: sh
+            args:
+            - -exc
+            - |
+              echo ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload' > ./mfe-app-learning/.env
+              echo BASE_URL='https://mitx-qa-next.mitx.mit.edu' >> ./mfe-app-learning/.env
+              echo CSRF_TOKEN_API_PATH='/csrf/api/v1/token' >> ./mfe-app-learning/.env
+              echo LMS_BASE_URL='https://mitx-qa-next.mitx.mit.edu' >> ./mfe-app-learning/.env
+              echo LOGIN_URL='https://mitx-qa-next.mitx.mit.edu/login' >> ./mfe-app-learning/.env
+              echo LOGOUT_URL='https://mitx-qa-next.mitx.mit.edu/logout' >> ./mfe-app-learning/.env
+              echo MARKETING_SITE_BASE_URL='https://mitx-qa-next.mitx.mit.edu' >> ./mfe-app-learning/.env
+              echo REFRESH_ACCESS_TOKEN_ENDPOINT='https://mitx-qa-next.mitx.mit.edu/login_refresh' >> ./mfe-app-learning/.env
+              echo SEARCH_CATALOG_URL='https://mitx-qa-next.mitx.mit.edu/courses' >> ./mfe-app-learning/.env
+              echo SITE_NAME='MITx QA' >> ./mfe-app-learning/.env
+              echo STUDIO_BASE_URL='https://studio-mitx-qa-next.mitx.mit.edu' >> ./mfe-app-learning/.env
+              echo USER_INFO_COOKIE_NAME='edx-user-info' >> ./mfe-app-learning/.env
```

Comment:
It's probably worth revisiting how we manage the environment variables during the pipeline execution now that we've figured out the mechanical aspects.